core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -1790,11 +1792,9 @@ public boolean isStopped() {
 
 	private String sendResultToSender(Message result) {
```

Comment:
De aanroeper van deze method heeft al een PipeLineSession.
Waarom kan die niet worden herbruikt?

Reply:
Omdat het hier gaat om het result, en die sessie is mogelijk al gesloten.

Reply:
Ik zie het, bij deze methode zou het kunnen ja

Reply:
Ik zie dat het op 29 Sept 2022 aangepast is, en dat hij daar altijd al een sessie had... https://github.com/frankframework/frankframework/commit/19c7f68a6ec3ae2ec99cefe4f6875cce1a94f0c3#diff-1d930132517ee5fb1b65086351c6359878618a44ce43037af8c4b46713960937R1773

Reply:
En zonder te kijken vermoed ik, dat ik degene ben die het heeft aangepast 🙈

Reply:
See issue #3762. Resolved as wont-fix, with the modification that the sender is called without a session.